deploy/operator-non-root.yaml

```
@@ -0,0 +1,89 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: security-operators
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: seccomp-operator
+  namespace: security-operators
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: config-map-reader
+  namespace: security-operators
+rules:
+- apiGroups: [""]
+  resources: ["configmaps"]
+  verbs: ["get", "watch", "list"]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: config-map-reader-binding
+  namespace: security-operators
+subjects:
+- kind: ServiceAccount
+  name: seccomp-operator
+  namespace: security-operators
+roleRef:
+  kind: Role
+  name: config-map-reader
+  apiGroup: rbac.authorization.k8s.io
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: seccomp-operator
+  namespace: security-operators
+spec:
+  selector:
+    matchLabels:
+      name: seccomp-operator
+  template:
+    metadata:
+      annotations:
+        seccomp.security.alpha.kubernetes.io/pod: 'runtime/default'
+      labels:
+        name: seccomp-operator
+    spec:
+      serviceAccountName: seccomp-operator
+      initContainers:
+        - name: non-root-enabler
+          image: busybox 
+          # Creates folder /var/lib/kubelet/seccomp/operator and sets 2000:2000 as its owner.
+          # This is required to allow the main container to run as non-root.
+          command: ['sh', '-c', 'if [ ! -d "/var/lib/kubelet/seccomp/operator" ]; then /bin/mkdir -p /var/lib/kubelet/seccomp/operator && chmod 0744 /var/lib/kubelet/seccomp/operator && /bin/chown -R 2000:2000 /var/lib/kubelet/seccomp; fi']
```

I suggest using somthing like:
```suggestion
          command:
            - sh -c
            - |
              if [ …]; then …
```